packages/terra-search-field/src/SearchField.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import classNames from 'classnames';
```

Comment:
Each component should import the base styles:
https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L4

Here's a little more information on why: https://github.com/cerner/terra-core/pull/251